### ISSUE TITLE
Where did the configurationFactory go in Webpack 5?

### ISSUE THREAD

**Issue by zhangxuyang950313** [DefinitelyTyped/DefinitelyTyped#51091]
Is something wrong with CI or publishing infrastructure?

- Check the [Infrastructure status updates](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/44317) thread for known issues.
- Leave a message on the [DefinitelyTyped Gitter](http://gitter.im/DefinitelyTyped/DefinitelyTyped).

If you know how to fix the issue, make a pull request instead.

- [ ] I tried using the `@types/xxxx` package and had problems.
- [ ] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [ ] I understand that `@types` package versions [do not follow SemVer](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library).
- [ ] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [ ] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @....

If you do not mention the authors the issue will be ignored.

**Comment by dopry** [comment]
Webpack 5 includes it's own types and you no longer need to use an @types package to go with it.

**Comment by jcharlesworthuk** [comment]
> Webpack 5 includes it's own types and you no longer need to use an @types package to go with it.

Webpack 5 does include its own types but ConfigurationFactory is not one of them, see: https://github.com/webpack/webpack/pull/11964

TLDR; just export a function from your webpack.ts

```
const createConfig = (env: any, argv: any) => { ... }
export default createConfig;
```
